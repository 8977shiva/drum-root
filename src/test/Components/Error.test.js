import React from "react";
import { shallow } from "enzyme";
import Error from "../../Components/Error";

describe("Error component", () => {
  test("should render properly", () => {
    const wrapper = shallow(<Error {...{ error: { statusCode: 404 } }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
