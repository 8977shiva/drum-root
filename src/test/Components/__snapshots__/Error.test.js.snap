// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render properly 1`] = `
<Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Not Found
    </title>
  </HelmetWrapper>
  <h2>
    Not Found
  </h2>
  <h3>
    Requested file was not found
  </h3>
</Fragment>
`;
